packages/apps/graph/src/graph/objects/block.ts

```diff
@@ -131,6 +129,47 @@ export default builder.prismaNode('Block', {
       },
     }),
 
+    events: t.prismaConnection({
+      type: 'Event',
+      cursor: 'blockHash_orderIndex_requestKey',
+      edgesNullable: false,
+      complexity: (args) => ({
+        field: getDefaultConnectionComplexity({
+          first: args.first,
+          last: args.last,
+        }),
+      }),
+      select: {
+        hash: true,
+      },
```

Comment:
Why don't we include events here? That way we can access it on the resolver via the parent object.